### PR TITLE
feat(console): add clean cache command

### PR DESCRIPTION
**Pull request by stonebuzz**
Add new command "clean cache"

```
dev  ~/dev/GLPI/9.5-bugfixes   add_clean_cache_command  php bin/console glpi:system:clean_cache        
cleaning cache done !!
 dev  ~/dev/GLPI/9.5-bugfixes   add_clean_cache_command  
```

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #number

### CODE REVIEW

File: inc/console/system/cleancachecommand.class.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      $this->setAliases(['system:clean_cache']);
+      $this->setDescription('Clean GLPI cache.');
+   }
+
+   protected function execute(InputInterface $input, OutputInterface $output) {
+
+      global $GLPI_CACHE;
+      $GLPI_CACHE->clear();
+
+      $output->writeln('<comment>cleaning cache done !!</comment>');
```
Comment (cedric-anne) [inline inc/console/system/cleancachecommand.class.php:59]:
```suggestion
      $output->writeln('<info>'. __('Cache cleared successfully.') . '</info>');
```

**Review by cedric-anne — COMMENTED** [review]
Although the word `clean` is understandable, `clear` seems more appropriate (change impact  class name, command name, alias and description).

File: inc/console/system/cleancachecommand.class.php (hunk 1 above)
Comment (trasher) [reply inc/console/system/cleancachecommand.class.php:59]: There already is a string in core "Cache reset successful"

Comment (cedric-anne) [reply inc/console/system/cleancachecommand.class.php:59]: I updated my suggestion

**Review by cedric-anne — APPROVED** [review]
I added a commit to apply my suggestions.

**Review by trasher — APPROVED** [review]

**Pull request merged**